### ISSUE TITLE
Zeit: Zweistufenmodell

### ISSUE THREAD

**Issue by freifrauvonbleifrei** [freifrauvonbleifrei/geburtstag#2]
Ein ganzes Wochenende zusammen zu verbringen, hat sowohl an der Mühle als auch im fabulösen erlanger Mechatronika-Freundeskreis Tradition ;) alle anderen sind natürlich auch dazu eingeladen.

Wem das zu viel des Guten ist, kann einfach am Samstag Abend "nur" zum feiern vorbeischauen. 

Also könnte der Plan ungefähr so aussehen:

|            	| Freitag 19                     	| Samstag 20            	| Sonntag 21                	|
|------------	|--------------------------------	|--------------------------------	|--------------------------------	|
| Früh       	|                                	| Meditation, Frühstück 	| Meditation, Frühstück     	|
| Mittag     	|                                	| Draußen sein, Picnic  	| Aufräumen, Reste essen    	|
| Nachmittag 	|                                	| Draußen               	| Kaffeetrinken mit Familie 	|
| Abends     	| 18:00 Ankommen und Abend essen 	| 19:00 Partey          	|                           	|
| Nacht      	| Spiele                         	| mehr Partey           	|                           	|

... das kommt darauf an, auf was ihr so Lust habt und anbietet (siehe andere Issues)

**Comment by Fellerer** [comment]
Und 20. auf 21. über geht nicht? Kann nämlich vermutlich erst zum 20. Nachmittag anreisen.

**Comment by freifrauvonbleifrei** [comment]
doch, geht auch :) aber da sind auch die schlafplätze am begehrtesten: https://github.com/freifrauvonbleifrei/geburtstag/projects/1 - ich hab dich mal eingeplant ;)
